### PR TITLE
Project 3 - Frida

### PR DESCRIPTION
**Pull request by fridamaria**
I enjoyed the assignment this week, some things are starting to feel a bit more intuitive and I feel I have a greater understanding of what I am doing.

However, JavaScript is tricky. I look forward to learning more!

I had some issues with getting the accordion animation to work, but got some help by a team member who posted an easy solution.

Link to site: https://pouroverpurist.netlify.com/

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,30 +1,162 @@
...
+    <section class="sign-up" id="sign-up">
+      <form class="contact-form" action="https://httpbin.org/anything" method="POST">
+        <h2>Sign Up!</h2>
+
+        <label>Name
+          <input type="text" name="name" placeholder="Enter your name" required>
+        </label>
+
+        <label>Email
+          <input type="text" name="email" placeholder="Enter your email address" required>
```
Comment (ericaglimsholt) [inline code/index.html]: There's a type called email actually! 
```suggestion
          <input type="email" name="email" placeholder="Enter your email address" required>
```

File: code/index.html [hunk 2]
```diff
@@ -1,30 +1,162 @@
...
+
+  </section>
+
+  <footer>
+    <p>
+      Spring 2020 Technigo Boot Camp - Project 3
+    </p>
+
+    <p>
+      Video credit: Video by <a
```
Comment (ericaglimsholt) [inline code/index.html]: Nice that you have included credit!  🙌

File: code/index.html (hunk 1 above)
Comment (fridamaria) [reply code/index.html]: Oh, missed that one! Have updated it ☺

**Comment by fridamaria** [comment]
Thank you for your comments Erica 🎉

**Pull request closed without merging**